docs/source/index.rst

```diff
@@ -81,13 +81,30 @@ Topics in this section will help you get started with ExecuTorch.
 .. toctree::
    :glob:
    :maxdepth: 1
-   :caption: Getting Started
+   :caption: Usage
    :hidden:
 
    getting-started
-   export-overview
-   runtime-build-and-cross-compilation
-   getting-started-faqs
+   using-executorch-export
+   using-executorch-android
+   using-executorch-ios
+   using-executorch-cpp
+   using-executorch-troubleshooting
+
+.. toctree::
+   :glob:
+   :maxdepth: 1
+   :caption: Backend Delegates
+   :hidden:
+
+   native-delegates-executorch-xnnpack-delegate
+   native-delegates-executorch-coreml-delegate
+   native-delegates-executorch-mps-delegate
+   native-delegates-executorch-vulkan-delegate
+   native-delegates-executorch-arm-delegate
+   native-delegates-executorch-qualcomm-delegate
+   native-delegates-executorch-mediatek-delegate
+   native-delegates-executorch-cadence-delegate
 
 .. toctree::
```

Comment:
we have too many tutorials here

can we remove everything in Tutorials section, and gradually increase if necessary?

Reply:
Sounds good. I 100% agree on the tutorials. A lot of what's in there is really reference material and not a tutorial. I'm happy to remove them for now.